CHANGES.md

```diff
@@ -13,6 +13,7 @@
 - Fix crash while formatting expressions using the walrus operator in complex
   `with` statements (#4630)
 - Handle `# fmt: skip` followed by a comment at the end of file (#4635)
+- Disallow unwrapping tuples in an `as` clause (#4634)
```

Comment:
This makes it sound like we're removing a language feature, I'd word it as something like "Fix crash when a tuple appears in the `as` clause of a `with` statement".